docs/index.adoc

```diff
@@ -3030,13 +3031,42 @@ class SingleOptionValidationExample {
 }
 ----
 
+.Kotlin
+[source,kotlin,role="secondary"]
+----
+class SingleOptionValidationExample : Runnable {
```

Comment:
(Similar to the Java example, I suggest leaving out the `: Runnable` bit, to keep it as small as possible.)

Reply:
I agree with your proposal. PR was modified as suggested.